### ISSUE TITLE
feature request: ghostscript included in cli container

### ISSUE THREAD

**Issue by leevh** [docksal/docksal#362]
Hello, I was attempting to use imagemagick to produce image thumbnails of PDF files in drupal and found that I had to install ghostscript on the cli container for this to work. Perhaps this could be included in a future release?

Thanks for Docksal, it is awesome!

**Comment by lmakarov** [comment]
@leevh do you know which packages/configs are necessary for ghostscript?
Would it be just `sudo apt-get install ghostscript` or something else?

**Comment by leevh** [comment]
@lmakarov it was just that command and it worked after that. Thanks!

**Comment by lmakarov** [comment]
Looks like this will add an extra 50MB to the image (uncompressed).

```
docker@cli:/var/www$ sudo apt install ghostscript
Reading package lists... Done
Building dependency tree       
Reading state information... Done
The following extra packages will be installed:
  fonts-droid gsfonts libcupsfilters1 libcupsimage2 libgs9 libgs9-common libijs-0.35 libjbig2dec0 libpaper-utils libpaper1 poppler-data
Suggested packages:
  ghostscript-x poppler-utils fonts-japanese-mincho fonts-ipafont-mincho fonts-japanese-gothic fonts-ipafont-gothic fonts-arphic-ukai fonts-arphic-uming fonts-nanum
The following NEW packages will be installed:
  fonts-droid ghostscript gsfonts libcupsfilters1 libcupsimage2 libgs9 libgs9-common libijs-0.35 libjbig2dec0 libpaper-utils libpaper1 poppler-data
0 upgraded, 12 newly installed, 0 to remove and 28 not upgraded.
Need to get 12.9 MB of archives.
After this operation, 47.9 MB of additional disk space will be used.
```

Since both Acquia and Pantheon include ghostscript in their stack, Docksal should also have it out of the box.

**Comment by lmakarov** [comment]
Pushed to `docksal/cli:edge-php5` and `docksal/cli:edge-php7`. Will be released in `docksal/cli` v1.4 (Docksal v1.7 most likely).

**Comment by lmakarov** [comment]
This will be included in the upcoming 2.0 cli image release with Docksal 1.7.0.
Available now in the edge versions:

```
docksal/cli:edge-php7.2
docksal/cli:edge-php5.6
docksal/cli:edge-php7.1
docksal/cli:edge-php7.0
```
